packages/calcite-components/src/components/chip/chip.scss

```diff
@@ -5,12 +5,13 @@
  *
  * @prop --calcite-chip-background-color: Specifies the component's background color.
  * @prop --calcite-chip-border-color: Specifies the component's border color.
+ * @prop --calcite-chip-close-icon-color: Specifies the component's close element icon color.
  * @prop --calcite-chip-corner-radius: Specifies the component's corner radius.
- * @prop --calcite-chip-text-color: Specifies the component's text color.
  * @prop --calcite-chip-icon-color: Specifies the component's icon color.
- * @prop --calcite-chip-close-icon-color: Specifies the component's close element icon color.
+ * @prop --calcite-chip-select-icon-color-press: Specifies the component's selection element icon color when active.
+ * @prop --calcite-chip-select-icon-color-pressed: [deprecate] Use `--calcite-chip-select-icon-color-press`. Specifies the component's selection element icon color when active.
```

Comment:
Should say `[deprecated` instead of `[deprecate]`. May apply elsewhere

Reply:
double checked. Does not apply elsewhere